src/tribol/common/Parameters.hpp

```diff
@@ -97,6 +97,8 @@ enum ContactCase
 {
```

Comment:
It's hard for me to remember what's a contact case vs. a model vs. a mode vs. a method.  Could you add a short definition of what each of these is?

Reply:
Added. This adds clarification for sure.